src/services/base/notification.ts

```diff
@@ -0,0 +1,9 @@
+import { FetchType, jsonFetchWrapper } from "@/submodules/javascript-functions/basic-fetch";
+import { BACKEND_BASE_URI } from "./_settings";
+
+export const notificationEndpoint = `${BACKEND_BASE_URI}/api/v1/notification`;
+
+export function getNotificationsByUser(onResult: (result: any) => void) {
+    const finalUrl = `${notificationEndpoint}`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
```

Comment:
the query for getting all notifications can be part of the notifications route instead of being part of the projects route